### ISSUE TITLE
Highlight vertices on hover. 

### ISSUE THREAD

**Issue by jeanbellynck** [jeanbellynck/Cayley-Graph-Generator#97]
Finding a the vertex tooltip is practically impossible if you are a new user. It might be smart to highlight a vertex on hover to imply clickability.

**Comment by p4jo** [comment]
I agree that the current hover mechanic is far from optimal. You have to stay on the vertex for the paths from the identity to load. It might me better to have to click on it and have some text field where its name is displayed and where you can also search for an element by a word describing it!
